.env.dev

```diff
@@ -0,0 +1,9 @@
+DATA_FOLDER=data-dev
+
+FUSEKI_PORT=3030
+ENDPOINT=http://fuseki:3030
+
+RDF_DIFFER_API_PORT=3040
+
+FLASK_APP=rdf_differ.entrypoints.flask_app
```

Comment:
Why do we need to know as an env variable what the APP is? Isn't it a single app?